Model/Gateway.php

```diff
@@ -463,4 +468,57 @@ public function fixedProductTax($order){
 
        return $weeTaxAmount;
     }
+
+    /**
+     * @param AbstractItem $item
+     * @return DataObject
+     */
+    public function getBuyRequestObject(AbstractItem $item)
+    {
+        /** @var DataObject $request */
+        $request = $item->getBuyRequest();
+        if (!$request && $item->getQuoteItem()) {
+            $request = $item->getQuoteItem()->getBuyRequest();
+        }
+        if (!$request) {
+            $request = new DataObject();
+        }
+
+        if (is_array($request)) {
+            $request = new DataObject($request);
+        }
+
+        return $request;
+    }
+
+    /**
+     * @param AbstractItem $item
+     * @return bool
+     */
+    public function isSubscription(AbstractItem $item)
+    {
+        $buyRequest = $this->getBuyRequestObject($item);
+
+        return $buyRequest->getData('subscribe') === 'subscribe';
+    }
+    /**
+     * @param CartInterface $quote
+     *
+     * @return bool
+     */
+    public function validateQuote($quote): bool
```

Comment:
The readme mentions PHP 5.5 support but this will require PHP 7.0. Have the requirements changed but not been updated?

Reply:
I see that it's supposed to support 5.6 now, so this is defiantly an issue.

Reply:
Removed